packages/server-boilerplate/src/types.ts

```diff
@@ -3,76 +3,10 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { Request } from 'express';
-import { ParamsDictionary, Query } from 'express-serve-static-core';
-import { SessionModel, SessionType } from './models/Session';
-
 export type AccessPolicyObject = Record<string, string[]>;
 
-export interface SessionCookie {
-  id: string;
-  email: string;
-  reset?: () => void;
-}
-
-export type TupaiaRequestBody = Record<string, unknown>;
-
-export interface TupaiaRequest
-  extends Request<ParamsDictionary, unknown, TupaiaRequestBody, Query> {
-  sessionModel: SessionModel;
-  sessionCookie?: SessionCookie;
-  session?: SessionType;
-}
-
-export interface TupaiaResponseBody {
-  accessPolicy?: AccessPolicyObject;
-  error?: string;
-}
-
-export interface Credentials {
-  emailAddress: string;
-  password: string;
-  deviceName: string;
-}
```

Comment:
Most of these types either got moved to being defined inline with the code that consumes them, or removed all together